template.yml

```diff
@@ -82,7 +87,7 @@ Parameters:
   InstanceName:
     Type: String
     Description: Optional - Customise the EC2 instance Name tag
-    Default: "buildkite-agent"
+    Default: ""
```

Comment:
Noting change to default. I presume you're preferring the value from `UseStackNameForInstanceName`?